prismic-model/js/parts/body.js

```diff
@@ -144,6 +144,15 @@ export default {
           linkText: text('Button text'),
         },
       }),
+      resourceList: slice('Resource List', {
+        repeat: {
+          resource: {
+            title: heading('Title', 3),
```

Comment:
We seem to use heading `2` in most other places, what's the reason for this being `3`?

I'm not too close to the issue, but could this exist as a generic list type, with tags to indicate styling them differently? A resource list feels very specific to the content in the list, where we could just have something like `titledList`?

Reply:
The idea is to use this slice within the body copy of a page, which should already have an h2 that the slice sits under.

A resource was defined in #6010 as:
"...a blob of content comprised of three parts:

A title
A short amount of body text that can contain formatted text, e.g. bullets
A single call to action: linking to an internal page or an external page

Resources stack together in a list"

But I think maybe making the slice have a more generic name and making use of tags makes sense, I'll do that